client/app/telecom/pack/xdsl/access/comfortExchange/pack-xdsl-access-comfort-exchange.component.js

```diff
@@ -0,0 +1,12 @@
+import controller from './pack-xdsl-access-comfort-exchange.controller';
+import template from './pack-xdsl-access-comfort-exchange.html';
+
+const xdslAccessComfortExchange = {
+  template,
+  controller,
+  bindings: {
+    xdslId: '<',
+  },
+};
+
+export default xdslAccessComfortExchange;
```

Comment:
You can order alphabetically and export directly :)

```suggestion
export default {
  bindings: {
    xdslId: '<',
  },
  controller,
  template,
};
```